Code/monet_cyclegan/data_acquisition/augment.py

```diff
@@ -96,6 +96,20 @@ def augment_image(image: tf.Tensor) -> tf.Tensor:
 
     return image
 
+def augment_image_without_crop(image: tf.Tensor) -> tf.Tensor:
+    """Randomly crop, resize, rotate and/or flip an image.
+
+    Args:
+        image: The image to be augmented.
+
+    Returns:
+        The augmented image.
+    """
+    image = random_rotate(image)
+    image = random_flip(image)
+
+    return image
+
```

Comment:
We can just have options for the augment image function to include/exclude different augmentations such as crop.

Reply:
I added a crop parameter for this function in my latest commit.